common/app/metrics/FrontendMetrics.scala

```diff
@@ -160,7 +159,7 @@ final case class SamplerMetric(override val name: String, override val metricUni
   }
 
   def recordSample(sampleValue: Double, sampleTime: DateTime): Future[List[SampledDataPoint]] =
-    dataPoints.alter(SampledDataPoint(sampleValue, sampleTime) :: _)
+    dataPoints.alter(SampledDataPoint(sampleValue, sampleTime) :: dataPoints.get())
```

Comment:
I was curious if this function exhibited the same behaviour as `record` above (i.e. `_` results in an 'unbound placeholder parameter error'). After writing tests I confirmed this and believe we now add new sampled data points to the head of the list.